### ISSUE TITLE
how to understand and use class_weight correctly?

### ISSUE THREAD

**Issue by mengxia1994** [open-mmlab/mmsegmentation#2474]
An example: It appears  in config like below(a two class problem, [background, a class, b class])
**loss_decode=dict(type='CrossEntropyLoss', use_sigmoid=False, loss_weight=1.0, class_weight=[0.5, 1.0, 1.0])**
is it designed to solve the class-imbalance problem? 
For example, I have 1000 annotations of A class and 200 annotations of B class. Should I set  B coefficient five times of A coefficient?
And then how to set coefficient of background?
Does it affect learning rate? In my experience, we would better use small batch size and small learning rate to achieve higher performance and big GPU nums to achieve faster training speed.

**Comment by mengxia1994** [comment]
after I doing like this(set B coefficient five times of A coefficient), the performance of B become lower......

**Comment by MengzhangLI** [comment]
`I have 1000 annotations of A class and 200 annotations of B class.` I do not understand this case. The "annotation" is PIXEL or IMAGES which have many pixels?

And yes, the `class_weight` is designed to solve class (label of pixels) imbalance problems. Please check out this [doc](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/zh_cn/tutorials/training_tricks.md#%E7%B1%BB%E5%88%AB%E5%B9%B3%E8%A1%A1%E6%8D%9F%E5%A4%B1-class-balanced-loss).

**Comment by mengxia1994** [comment]
您好，我已收到您的邮件，我将尽快查阅并给予回复，谢谢！（孟夏的自动回复）

**Comment by mengxia1994** [comment]
> `I have 1000 annotations of A class and 200 annotations of B class.` I do not understand this case. The "annotation" is PIXEL or IMAGES which have many pixels?
> 
> And yes, the `class_weight` is designed to solve class (label of pixels) imbalance problems. Please check out this [doc](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/zh_cn/tutorials/training_tricks.md#%E7%B1%BB%E5%88%AB%E5%B9%B3%E8%A1%A1%E6%8D%9F%E5%A4%B1-class-balanced-loss).

my bad, the number means the quantity of each class
